redash/models/__init__.py

```diff
@@ -579,12 +579,25 @@ def outdated_queries(cls):
     @classmethod
     def search(cls, term, group_ids, user_id=None, include_drafts=False,
                limit=None, include_archived=False):
+        from redash.authentication.org_resolving import current_org
         all_queries = cls.all_queries(
             group_ids,
             user_id=user_id,
             include_drafts=include_drafts,
             include_archived=include_archived,
         )
+
+        if current_org.get_setting("multi_byte_search_enabled"):
+            # Since tsvector doesn't work well with CJK languages, use `ilike` too
+            pattern = u'%{}%'.format(term)
+            return all_queries.filter(
+                or_(
+                    cls.search_vector.match(term),
```

Comment:
Isn't it enough to match only `name` and `description`?

Reply:
I'm not sure it is popular, but `search_vector` contains the query's SQL too, so we can find by some terms in SQL with this (only with small overhead). How do you think?

Reply:
In this case, you can add the `query_text` column to the search instead. But I think it's better to leave it out, because for some search terms it can yield really bad results (like when you search for something that is also a table name). When using the regular search it has weights in place, so those are ranked lower.

Reply:
Thank you for your advise. I have removed `search_vector` for multi-byte search. It still provides reasonable search results for me.